wordpress-mobile/gutenberg-mobile
# Update Gutenberg ref to point to text-clipping fix attempt

Fixes #2136 

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/21532
WPAndroid PR to produce an instalable APK: https://github.com/wordpress-mobile/WordPress-Android/pull/11641

To test:
1. If using a device that didn't experience the issue, nothing should change
2. On a device that experiences the issue, the media placeholder link text will hopefully not get clipped. Try adding a Media&Text block to verify.

PR submission checklist:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. Comment: this fix is quite device specific and for an issue that's not too disruptive that, I don't think we need a release note entry.
